utils/generate_html_index_srg_mapping.sh

```diff
@@ -0,0 +1,46 @@
+#!/usr/bin/env bash
+# suppress stdout from pushd and popd commands
+pushd () {
+    command pushd "$@" > /dev/null
+}
+
+popd () {
+    command popd "$@" > /dev/null
+}
+
+if [ -z "$1" ]; then
+    echo "You must inform a directory to store built files."
+    exit 1
+fi
+PAGES_DIR=$1
+
+if [ -z "$2" ]; then
+    echo "You must inform a list of products in double quotes."
+    exit 1
+fi
+srg_products=$2 # space separated list of products
+
+mkdir -p $PAGES_DIR
+
+# Generate SRG Mapping Tables
+pushd $PAGES_DIR
+touch index.html
+echo "<html>" > index.html
```

Comment:
```suggestion
echo "<!DOCTYPE html>" > index.html
echo '<html lang="en">' >> index.html
echo '<meta charset="utf-8" />' >>
```
If we want this to be a valid HTML5 document.